Task_1/FeTS_Challenge.py

```diff
@@ -518,15 +518,18 @@ def FedAvgM_Selection(local_tensors,
 # to those you specify immediately above. Changing the below value to False will change 
 # this fact, excluding the three hausdorff measurements. As hausdorff distance is 
 # expensive to compute, excluding them will speed up your experiments.
-include_validation_with_hausdorff=True
+include_validation_with_hausdorff=True # IF SET TO FALSE WHOLE CODE RUNS SEGMENTATION FOR SOME REASON
```

Comment:
This is a bug, right? So I think it makes sense to open an issue to keep track of it.

Reply:
there is already an open issue for it: https://github.com/FeTS-AI/Challenge/issues/200